idea/resources-fir/META-INF/firIntentions.xml

```diff
@@ -4,5 +4,20 @@
       <className>org.jetbrains.kotlin.idea.fir.intentions.declarations.HLSpecifyExplicitTypeForCallableDeclarationIntention</className>
       <category>Kotlin</category>
     </intentionAction>
+
+    <intentionAction>
```

Comment:
Please add intention descriptions for those intentions (may be copied from FE10 implementation https://github.com/JetBrains/kotlin/tree/0df5d5158d4334de140434d3c2c8eab4318be208/idea/resources-en/intentionDescriptions/AddPropertyAccessorsIntention)